### PR TITLE
Add current date and microTime function.

### PR DESCRIPTION
**Pull request by niiyz**
I Add current data and microTime function.

### CODE REVIEW

File: tests/Carbon/Fixtures/MyCarbon.php [hunk 1]
```diff
@@ -4,7 +4,20 @@
...
+ * @package Tests\Carbon\Fixtures
+ */
 class MyCarbon extends Carbon
 {
-    //
+    /**
+     * Get a Carbon instance for the current date and microTime
+     * @return static
+     */
+    public static function currentMicroTime()
```
Comment (lucasmichot) [inline tests/Carbon/Fixtures/MyCarbon.php]: Why is this function not in `\Carbon\Carbon`?

File: tests/Carbon/MicrotimeTest.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Tests\Carbon;
+
+use Tests\Carbon\Fixtures\MyCarbon as Carbon;
+use Tests\AbstractTestCase;
+
+class MicrotimeTest extends AbstractTestCase
+{
+    public function testMicroTime()
```
Comment (lucasmichot) [inline tests/Carbon/MicrotimeTest.php]: This function should be in `\Tests\Carbon\ConstructTest`

**Comment by lucasmichot** [comment]
Please check your code style with `php-cs-fixer fix` or apply this patch https://styleci.io/analyses/8mKp7q

**Comment by GrahamCampbell** [comment]
> Please check your code style with php-cs-fixer fix or apply this patch https://styleci.io/analyses/8mKp7q

Note that running php-cs-fixer will often not give the same results as StyleCI.

**Comment by lucasmichot** [comment]
Ping @niiyz

**Pull request closed without merging**